projects/event-lambdas/src/event-api-lambda/application.ts

```diff
@@ -20,7 +20,13 @@ export const createApp = (initConfig: AppConfig): express.Application => {
       res.header("Access-Control-Allow-Methods", "OPTIONS,POST,GET");
       res.header("Access-Control-Allow-Credentials", "true");
     }
-    next();
+    if (req.method === "OPTIONS") {
+      res.header("Vary", "Origin")
+      res.header("Cache-control", "max-age=600")
```

Comment:
It could, yes. I read something somewhere that Chrome caps it at 10 minutes anyway. But I've just seen something else suggesting it is 2 hours. Does 24 hours seem reasonable to you?